app/assets/stylesheets/darkswarm/_shop-navigation.css.scss

```diff
@@ -49,7 +49,7 @@ ordercycle {
       border-radius: 0.25em 0 0 0.25em;
       float: left;
       font-size: 1em;
-      line-height: 1.5em;
+      line-height: 1.3em;
```

Comment:
I am not sure the line-height on the select does it's job, but I dont want to change it, so I just move both label and select to 1.3.
On my tests with firefox:
before
![image](https://user-images.githubusercontent.com/1640378/80871445-945efe00-8ca4-11ea-9943-341b9f061972.png)

after
![image](https://user-images.githubusercontent.com/1640378/80871456-a2148380-8ca4-11ea-9c54-8edeffe8f2d4.png)